packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -136,6 +136,16 @@ export abstract class BaseTargetGroup extends cdk.Construct implements ITargetGr
    */
   public readonly targetGroupName: string;
 
+  /**
+   * ARNs of load balancers load balancing to this TargetGroup
+   */
+  public readonly targetGroupLoadBalancerArns: string[];
+
+  /**
+   * Full name of first load balancer
```

Comment:
Can you add a comment why would that be useful to a user?

Reply:
It's mostly just useful for metrics and for autoscaling (which probably also uses it to access the metrics)